sdk/appconfiguration/azure-data-appconfiguration/src/samples/java/com/azure/data/appconfiguration/AadAuthentication.java

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.data.appconfiguration;
+
+import com.azure.core.credential.TokenCredential;
+import com.azure.data.appconfiguration.models.ConfigurationSetting;
+
+/**
+ * Sample demonstrates how to use AAD token to build a configuration client.
+ */
+public class AadAuthentication {
+    /**
+     * Sample for how to use AAD token Authentication.
+     *
+     * @param args Unused. Arguments to the program.
+     */
+    public static void main(String[] args) {
+        // The endpoint value can be obtained from connection string. Connection String has string format
+        // "endpoint={endpoint_value};id={id_value};secret={secret_value}" where the endpoint is what it is.
+        // The connection string can be obtained by going to your App Configuration instance in the Azure portal
+        // and navigating to "Access Keys" page under the "Settings" section.
```

Comment:
Same comment as the README, let's talk about accessing the portal when discussing getting endpoint for TokenCredential, at least make it more prominent and talk about it before connection strings.